README.md

```diff
@@ -19,88 +19,141 @@ Synopsys Security Scan supports Azure integration with the following Synopsys se
-    bridge_polaris_serverUrl: $(POLARIS_SERVER_URL)
-    bridge_polaris_accessToken: $(POLARIS_ACCESS_TOKEN)
-    bridge_polaris_application_name: $(POLARIS_APPLICATION_NAME)
-    bridge_polaris_project_name: $(POLARIS_PROJECT_NAME)
+    BRIDGE_POLARIS_SERVERURL: $(POLARIS_SERVER_URL)
+    BRIDGE_POLARIS_ACCESSTOKEN: $(POLARIS_ACCESS_TOKEN)
+    BRIDGE_POLARIS_APPLICATION_NAME: $(Build.Repository.Name)
+    BRIDGE_POLARIS_PROJECT_NAME: $(Build.Repository.Name)
     ### Accepts Multiple Values
-    bridge_polaris_assessment_types: "SCA,SAST"
+    BRIDGE_POLARIS_ASSESSMENT_TYPES: "SCA,SAST"
+    ### Uncomment below configuration if Synopsys Bridge diagnostic files needs to be uploaded
+    # INCLUDE_DIAGNOSTICS: true
 ```
+
+**Please find the following mandatory and optional paramters for Polaris below:**
+
 | Input Parameter            | Description                                                       | Mandatory / Optional | 
 |----------------------------|-------------------------------------------------------------------|--------------------|
-| `bridge_polaris_serverUrl`        | URL for Polaris Server                                            | Mandatory          |
-| `bridge_polaris_accessToken`      | Access token for Polaris                                          | Mandatory          |
-| `bridge_polaris_application_name` | Application name in Polaris                                       | Mandatory          |
-| `bridge_polaris_project_name`     | Project name in Polaris                                           | Mandatory          |
-| `bridge_polaris_assessment_types` | Polaris assessment types. Example: SCA,SAST                       | Mandatory          |
+| `BRIDGE_POLARIS_SERVERURL`        | URL for Polaris Server                                            | Mandatory          |
+| `BRIDGE_POLARIS_ACCESSTOKEN`      | Access token for Polaris                                          | Mandatory          |
+| `BRIDGE_POLARIS_APPLICATION_NAME` | Application name in Polaris                                       | Mandatory          |
+| `BRIDGE_POLARIS_PROJECT_NAME`     | Project name in Polaris                                           | Mandatory          |
+| `BRIDGE_POLARIS_ASSESSMENT_TYPES` | Polaris assessment types. Example: SCA,SAST                       | Mandatory          |
 
 # Synopsys Security Scan - Coverity Cloud Deployment with Thin Client
 
 At this time, Synopsys Security Scan only supports the Coverity thin client/cloud deployment model, which removes the need for a large footprint installation in your agent.
 
 Before running Coverity using the Synopsys Security Scan, ensure the appropriate `project` and `stream` are set in your Coverity Connect server environment.
 
-Configure sensitive data like usernames, passwords and URLs using pipeline variables.
+Synopsys Security Scan Extension available in the Azure DevOps Marketplace is the recommended solution for integrating Coverity into azure pipeline.
```

Comment:
azure should be Azure. check for other places as well.

Reply:
updated